README.md

```diff
@@ -0,0 +1,105 @@
+# `mut`
+
+https://github.com/mongodb/mut
+Copyright 2023 MongoDB Inc.
+
+`mut` provides a handful of tools used in MongoDB's documentation platform.
+
+* `mut-index` turns snooty-parser-generated abstract syntax trees into manifests
+  that can be ingested into Atlas Search to power the docs' search.
+* `mut-redirects` generates redirects from our bespoke redirect definition format,
+  making it "easy" to ensure that readers never find themselves on a 404ing page
+  when swapping versions or following old links
+* `mut-images` turns SVGs into PNGs I think?
+* `mut-stage` uploads files to S3 with minimal fuss for the user
+
+`mut` is licensed under the Apache License, Version 2.0. 
+See the `LICENSE` file for details.
+
+## Installation
+
+### Legacy mut (v0.9 and earlier)
+
+`mut v0.9.x` exists to support the needs of MongoDB docs properties building with the legacy (giza)
+toolchain. To use `mut`, you need python3, along with a bunch of other dependencies.
+To install `mut` for use with giza, follow the instructions on the writer setup wiki.
+
+### Modern `mut` (v0.10 and later)
+
+To use mut, you need python 3.8 or later.
+
+As of v0.10, each `mut` [release](https://github.com/mongodb/mut/releases) includes a python wheel 
```

Comment:
This isn't yet true for v0.10.3 -- we could add it to the github CI pipeline so it gets made automatically!